server/stt/stt_planning_ml.py

```diff
@@ -124,7 +136,221 @@ async def get_coverage_details(
+                    "sttworkstatus:5": "ncostat:notdec",
+                }
+
+                mapped_status = status_mapping.get(qcode)
+                if mapped_status:
+                    coverage["news_coverage_status"] = {
+                        "qcode": mapped_status,
+                        "name": self.get_coverage_status_name(mapped_status),
+                        "label": self.get_coverage_status_label(mapped_status),
+                    }
+                break
+
+    def get_coverage_status_name(self, qcode):
+        """Get coverage status name from qcode"""
+        names = {
+            "ncostat:int": "coverage intended",
+            "ncostat:notint": "coverage not intended",
+            "ncostat:notdec": "coverage not decided yet",
+        }
+        return names.get(qcode, "")
+
+    def get_coverage_status_label(self, qcode):
+        """Get localized Finnish label for coverage status"""
+        labels = {
+            "ncostat:int": "Kyllä",
+            "ncostat:notint": "Ei",
+            "ncostat:notdec": "Ei",
+        }
+        return labels.get(qcode, "")
+
+    def parse_coverage_internal_note(
+        self, planning_elt: Element, coverage: Dict[str, Any]
+    ):
+        """Parse internal note for coverage from sttinternaltext subject"""
+        # Find subject with qcode sttinternaltext
+        for subject_elt in planning_elt.findall(self.qname("subject")):
+            if subject_elt.get("qcode") == "sttinternaltext":
+                value_elt = subject_elt.find(self.qname("value"))
+                if value_elt is not None and value_elt.text:
+                    coverage["internal_note"] = value_elt.text
+                break
+
+    def parse_picture_type(self, planning_elt: Element, coverage: Dict[str, Any]):
+        """Parse picture type from genre and sttimagetypename subject"""
+        # First try genre element
+        genre_elt = planning_elt.find(self.qname("genre"))
+        if genre_elt is not None:
+            qcode = genre_elt.get("qcode")
+            if qcode and qcode.startswith("sttimage:"):
+                coverage.setdefault("planning", {})["sttimagetype"] = qcode
+                return
+
+        # Fallback to sttimagetypename subject
+        for subject_elt in planning_elt.findall(self.qname("subject")):
+            qcode = subject_elt.get("qcode", "")
+            if qcode.startswith("sttimagetypename:"):
+                qcode = qcode.replace("sttimagetypename:", "sttimage:")
+                coverage.setdefault("planning", {})["sttimagetype"] = qcode
+                break
+
+    def parse_photographer_awareness(
+        self, planning_elt: Element, coverage: Dict[str, Any]
+    ):
+        """Parse photographer awareness from sttphotoaware subject"""
+        for subject_elt in planning_elt.findall(self.qname("subject")):
+            qcode = subject_elt.get("qcode", "")
+            if qcode.startswith("sttphotoaware:"):
```

Comment:
This should use the [sttdoesphotographerknow](https://github.com/superdesk/superdesk-stt/blob/develop/server/data/vocabularies.json#L856) CV, and stored in Coverage custom cv.

If the value is `sttphotoaware:0`, then we don't store this value at all